Packs/DeCYFIR/Integrations/decyfir/decyfir_description.md

```diff
@@ -0,0 +1,15 @@
+ # Partner Contributed Integration
+Integration Author: <b> Cyfirma Team</b> 
+
+#### DeCYFIR API's provides External Threat Landscape Management insights. 
+
+Support and maintenance for this integration are provided by the author. Please use the following contact details:
+
+    Email: contact@cyfirma.com,
+    URL: https://www.cyfirma.com
```

Comment:
Please add to secrets_ignore file.

Reply:
We corrected this.  we can push next final commit.